### DISCUSSION TITLE
Welcome to mill Discussions!

### DISCUSSION THREAD

**Discussion by lefou** [com-lihaoyi/mill#1073] [General]
## 👋 Welcome!
We're now also using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. 

Remember that this is a community we build together 💪.

Our existing [mill Gitter room](https://gitter.im/lihaoyi/mill) will be still a place to chat.

**Comment by tballard** [comment]
I was trying to run a copy of mill that I could add debug to so that I could track down issues. Having built a mill assembly jar, I was trying to run it using:
```
JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk-12.0.1.jdk/Contents/Home \
$JAVA_HOME/bin/java -DMILL_CLASSPATH=/Users/tballard/git/mill-master/out/main/assembly/dest/out.jar \
-DMILL_VERSION=0.8.0 \
-Djna.nosys=true \
-cp /Users/tballard/git/mill-master/out/main/assembly/dest/out.jar mill.main.client.MillClientMain \
turbine.compile
```
I came up with that by trying to track down what mill normally ends up doing. Didn't work though. Although the build.sc normally works, it gets a bunch of stuff like 
```
build.sc:8: object scalalib is not a member of package mill
import mill.scalalib._
            ^
build.sc:35: not found: type MavenModule
trait RedsproModule extends MavenModule with PublishModule {
                            ^
build.sc:35: not found: type PublishModule
trait RedsproModule extends MavenModule with PublishModule {
```

I'd greatly appreciate a hint.

**Reply by lefou** [reply]
Welcome!

Instead of `main.assembly` you might want try the outcome of `dev.assembly`, as that is what we use as mill assembly.

(Also, next time, please open a new discussion thread if you are going to start a new topic, thanks.)

**Reply by tballard** [reply]
Hey, it worked. Yay!
